packages/components/atoms/f-form-field/README.md

```diff
@@ -1,10 +1,11 @@
+<div style="text-align: center">
```

Comment:
Looking at the preview, this section is no longer centre-aligned following this change?

Reply:
I think this needs to stay as `align="center"` to keep gh-pages markdown happy.

Reply:
@DamianMullins @ashleynolan 

Fixed.  Could you have another look?

Reply:
More out of interest than anything else – what's the benefit to using `style="text-align: center"` over `align="center"`?

I'm a bit averse to inline styles unless they're absolutely necessary.

Reply:
I'm not sure centre-alignment actually works with the updated syntax from what I can remember when this went in originally.

Reply:
I would stick to the old syntax tbh, unless there's a need to change it. It's just holding up the rest of the changes getting merged and I don't think it really adds anything to change it?